FogCreek/Glitch-Community
# Hungry pipe: fix super user button styling

## Links
* https://hungry-pipe.glitch.me/

## GIF/Screenshots:
before:
<img width="176" alt="Screen Shot 2019-08-02 at 3 32 17 PM" src="https://user-images.githubusercontent.com/6620164/62394294-c509df80-b53a-11e9-87d6-ee461d68524c.png">
after:
<img width="178" alt="Screen Shot 2019-08-02 at 3 32 27 PM" src="https://user-images.githubusercontent.com/6620164/62394295-c509df80-b53a-11e9-8145-c09805482159.png">

## Changes:
* add some padding around the super user toggle button

## How To Test:
* if you have the ability to be a super user you can click on your user dropdown on the top right part of the page and see that it looks correct (if you don't have the ability to be a super user, let me know and we can work out how to get that set for you to test).
